lib/ace/virtual_renderer.js

```diff
@@ -662,7 +662,6 @@ var VirtualRenderer = function(container, theme) {
             if (composition.useTextareaForIME) {
                 var val = this.textarea.value;
                 w = this.characterWidth * (this.session.$getStringScreenWidth(val)[0]);
-                h += 2;
```

Comment:
not sure why we need to add 2 pixels here? :)

Reply:
this was needed before when textarea had padding 1px, but the padding was removed some time ago causing textarea to be misaligned during composition, so i am removing the +2.